### PR TITLE
doc(effects): add note on initializing effects

### PR DESCRIPTION
**Pull request by bcabanes**
Hi!

This add some info on how to run some code when the effect is loaded, in the readme.

Have a great day!

### CODE REVIEW

File: docs/effects/api.md [hunk 1]
```diff
@@ -93,6 +93,51 @@ export class SomeEffectsClass {
...
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+  
+  @Effect() authActions$ = this.action$.pipe(
+      ofType<LoginAction | LogoutAction>('LOGIN', 'LOGOUT'),
+      tap(action => console.log(action))
+    );
+
+  @Effect() otherAction$ = this.action$.pipe(
```
Comment (brandonroberts) [inline docs/effects/api.md]: Remove this effect from the example

File: docs/effects/api.md [hunk 2]
```diff
@@ -93,6 +93,51 @@ export class SomeEffectsClass {
...
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { defer } from 'rxjs/observable/defer';
+import { LoginAction, LogoutAction } from './auth';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+  
+  @Effect() authActions$ = this.action$.pipe(
```
Comment (brandonroberts) [inline docs/effects/api.md]: Add a `{ dispatch: false }` here, or the effect will error if you copy/paste this code as-is.

Comment (brandonroberts) [reply docs/effects/api.md]: Fix the spacing in the code snippet to be 2 spaces.

File: docs/effects/api.md [hunk 3]
```diff
@@ -93,6 +93,51 @@ export class SomeEffectsClass {
...
+import { Injectable } from '@angular/core';
+import { Actions, Effect, ofType } from '@ngrx/effects';
+import { defer } from 'rxjs/observable/defer';
+
+@Injectable()
+export class SomeEffectsClass {
+  constructor(private actions$: Actions) { }
+
+  @Effect({ dispatch: false }) init$: Observable<any> = defer(() => {
+    return of(console.log('init$'));
```
Comment (brandonroberts) [inline docs/effects/api.md]: just use `tap` and `console.log` here. Returning an observable of the console log seems a bit confusing.

**Comment by bcabanes** [comment]
@brandonroberts Thanks! PR updated!

**Comment by brandonroberts** [comment]
Thanks!

**Pull request merged**